README.md

```diff
@@ -18,7 +18,10 @@ Cocotb requires:
 
 - Python 3.5+
 - A C++11 compiler
-- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/))
+- An HDL simulator (such as [Icarus Verilog](http://iverilog.icarus.com/),
+[Verilator](https://www.veripool.org/wiki/verilator),
```

Comment:
I think the reason we did not include these explicitly in this list is that Icarus is the only open-source simulator that actually passes all our tests.

Reply:
OTOH, we also have some ``expected_error`` markers for Icarus Verilog. I guess we could add a note that not every simulator supports everything, and that information gets more useful once we link from Supported Simulators to the respective labelled issues.

Reply:
> once we link from Simulator Support to the respective labelled issues.

Opened PR #1979 for this.